### ISSUE TITLE
[Suggestion] make the texture be able to have various names.

### ISSUE THREAD

**Issue by Kobins** [JannisX11/blockbench#772]
I'm implementing gun animation by playing several models in a row.
I'm using the `parent` function because multiple model definitions increase capacity.
However, with the `parent` feature, if the model changes, the file size increases unnecessarily because the `elements` part needs to be newly defined.
To solve this problem, we wanted to create a basic model that contained all of the model forms, of which only textures could be inherited and used as needed in each submodel. This allows you to implement multiple models with just one model definition.

Let me explain the shape of the gun with or without the magazine.
To separate textures in the magazine section, you can either separate textures with a new file, or **assign a variable name pointing to the same texture file**, so that sub-models apply transparent textures to that variable.

However, since BlockBench handles textures based on texture files (paths), BlockBench seems to integrate them into one if there are multiple variables that point to the same texture file.

It doesn't matter when modifying the json file directly, but I suggest adding the function because I think there will be a lot of difficulties without BlockBench in the process of importing the attachment afterwards.

**Comment by JannisX11** [comment]
Blockbench can edit these parent files without issues, but I agree that there could be a feature to create a dummy texture with a # name.

**Comment by Luke18033** [comment]
That's not what this was suggesting...? It should be possible to have multiple different texture variables which point to the same texture, currently Blockbench will switch everything to use the first of these variables and remove the others when loading the model.

**Comment by Dhranios** [comment]
I see what Luke means, for example, I have an item with a clock option, which uses textures to show the numbers, each number has a different texture variable, but the default model will have them all hidden, what blockbench will do is make all the elements that use the texture (even if they have different variable names) the same variable, effectively breaking the model as soon as I open the model in blockbench, as all child files will no longer work then.

Here:
```		"screen": "luigis_mansion:item/gameboy_horror/screen",
		"time_apm": "luigis_mansion:item/empty",
		"time_colon": "luigis_mansion:item/empty",
		"hour_1": "luigis_mansion:item/empty",
		"hour_2": "luigis_mansion:item/empty",
		"minute_1": "luigis_mansion:item/empty",
		"minute_2": "luigis_mansion:item/empty"```
What blockbench will do is make all elements that use "empty" time_apm variables, and delete the other variables.

**Comment by Luke18033** [comment]
Yep. It seems like Blockbench is keeping track of textures based on the filename, with each one just having a variable associated with it. It should be the other way around.
